calm/draft/2024-03/meta/core.json

```diff
@@ -0,0 +1,229 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/finos-labs/architecture-as-code/main/calm/draft/2024-03/meta/core.json",
+  "title": "Common Architecture Language Model (CALM) Vocab",
+  "properties": {
+    "nodes": {
+      "type": "array",
+      "items": {
+        "$ref": "#/defs/node"
+      }
+    },
+    "relationships": {
+      "type": "array",
+      "items": {
+        "$ref": "#/defs/relationship"
+      }
+    },
+    "required": [
+      "nodes",
+      "relationships"
+    ]
+  },
+  "defs": {
+    "node": {
+      "type": "object",
+      "properties": {
+        "uniqueId": {
+          "type": "string"
+        },
+        "node-type": {
+          "$ref": "#/defs/node-type-definition"
+        },
+        "name": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "data-classification": {
+          "$ref": "#/defs/dataClassification"
+        },
+        "run-as": {
+          "type": "string"
+        },
+        "instance": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "uniqueId",
+        "node-type",
+        "name",
+        "description"
+      ],
+      "additionalProperties": true
+    },
+    "relationship": {
+      "type": "object",
+      "properties": {
+        "uniqueId": {
+          "type": "string"
+        },
+        "relationship-type": {
+          "type": "object",
+          "properties": {
+            "interacts": {
+              "$ref": "#/defs/interacts-type"
+            },
+            "connects": {
+              "$ref": "#/defs/connects-type"
+            },
+            "deployed-in": {
+              "$ref": "#/defs/deployed-in-type"
+            },
+            "composed-of": {
+              "$ref": "#/defs/composed-of-type"
+            }
+          },
+          "oneOf": [
+            {
+              "required": [
+                "deployed-in"
+              ]
+            },
+            {
+              "required": [
+                "composed-of"
+              ]
+            },
+            {
+              "required": [
+                "interacts"
+              ]
+            },
+            {
+              "required": [
+                "connects"
+              ]
+            }
+          ]
+        },
+        "protocol": {
+          "$ref": "#/defs/protocol"
+        },
+        "authentication": {
+          "$ref": "#/defs/authentication"
+        }
+      },
+      "required": [
+        "uniqueId",
+        "relationship-type"
+      ],
+      "additionalProperties": true
+    },
+    "dataClassification": {
```

Comment:
@aidanm3341 when you create the issue for `unqiue-id` it's worth including `data-classification too`

cc/ @rocketstack-matt

Reply:
Already have #45